app/models/search_result.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class SearchResult < ApplicationRecord
+  enum search_engines: { google: 0, bing: 1 }
```

Comment:
Also prefer to keep naming to singular to avoid typos and guesses:

```suggestion
  enum search_engine: { google: 0, bing: 1 }
```

FYI there are two syntax you pick from: https://blog.saeloun.com/2021/02/26/rails-introduces-new-syntax-for-enum/#after